README.md

```diff
@@ -1,3 +1,45 @@
-# cr-hello-microverse
-Hello microverse project
+# CR Practice project - Hello Microverse
 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/Academic-blue)
+![](https://img.shields.io/badge/HTML-red)
+![](https://img.shields.io/badge/SCSS-ff69b4)
+![](https://img.shields.io/badge/JavaScript-yellow)
+
+
+## Learning objectives
+
+- This project was built with the objective of practicing code reviews.
+
+## Author
+
+👤 *Gabriel Rendon*
+
+- GitHub: [@gabrielyea](https://github.com/gabrielyea)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+<!--Feel free to check the [issues page](../../issues/).-->
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+<!--## Acknowledgments-->
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check and add issues to the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Please fix this broken link! Make sure the link to `./MIT.md` exists, this allows you to be professional!